README.md

```diff
@@ -1,80 +1,90 @@
 # multi-cluster-traffic-controller
-// TODO(user): Add simple overview of use/purpose
 
-## Description
-// TODO(user): An in-depth paragraph about your project and overview of use
-
-## Getting Started
-You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
-**Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
-
-### Running on the cluster
-1. Install Instances of Custom Resources:
-
-```sh
-kubectl apply -f config/samples/
-```
-
-2. Build and push your image to the location specified by `IMG`:
-	
-```sh
-make docker-build docker-push IMG=<some-registry>/multi-cluster-traffic-controller:tag
-```
+## Description:
+When deploying the multi cluster traffic controller using the make targets the following will be created: 
+* Kind cluster(s)
+* Gateway API CRDs in the control plane cluster
+* Ingress controller
+* Cert manager
+* External DNS
+* Webhook and webhook config
+* ARGO CD instance
+* K8s Dashboard
+* Lets encrypt certs
 	
-3. Deploy the controller to the cluster with the image specified by `IMG`:
-
-```sh
-make deploy IMG=<some-registry>/multi-cluster-traffic-controller:tag
-```
-
-### Uninstall CRDs
-To delete the CRDs from the cluster:
-
-```sh
-make uninstall
-```
-
-### Undeploy controller
-UnDeploy the controller to the cluster:
-
-```sh
-make undeploy
-```
-
-## Contributing
-// TODO(user): Add detailed information on how you would like others to contribute to this project
-
-### How it works
-This project aims to follow the Kubernetes [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
-
-It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/) 
-which provides a reconcile function responsible for synchronizing resources untile the desired state is reached on the cluster 
-
-### Test It Out
-1. Install the CRDs into the cluster:
-
-```sh
-make install
-```
-
-2. Run your controller (this will run in the foreground, so switch to a new terminal if you want to leave it running):
-
-```sh
-make run
-```
-
-**NOTE:** You can also run this in one step by running: `make install run`
-
-### Modifying the API definitions
-If you are editing the API definitions, generate the manifests such as CRs or CRDs using:
 
-```sh
-make manifests
-```
 
-**NOTE:** Run `make --help` for more information on all potential `make` targets
+## Prerequisites:
+* AWS
+* Kind 
+    * `make kind`
+* yq 
+    * `make yq`
```

Comment:
Why yq?

Reply:
its needed for the webhook config to inject the ca bundle

Reply:
This is fine. I would say though that when i see yq or jq being added a dependency of a deploy, it usually tells me we are doing something wrong. Ideally the kustomizations that we are using should override anything required without manipulating yaml directly.